firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -768,6 +763,22 @@ def get_doc_id(document_pb, expected_prefix):
     return document_id
 
 
+def get_field_paths(update_data):
```

Comment:
I named this `extract_field_paths` in the other SDKs. I think that's a much more descriptive name.

Furthermore, `update_data` is something different in the Firestore SDKs. Updata data is 
`{ fieldPath1 : value, fieldPath2: value }`. I think you are just trying to iterate a map of data here `{ field : value, field: value }`. That's `document_data`.

Reply:
I'm actually recursing the `get_field_paths` (see line 774), so I want the actual `fieldPath` to the value as an api_repr.  So I think this is actually an `update_data`?

I think I wrote another method to extract both, but for this PR and what I needed here, the field path itself suffices.  

It seems like there are a few ways to represent the same thing:  In dictionary form `{'a': {'b': {1.2 : value}}}`, in api_repr form `{'a.b.(bt)1.2(bt) : value}`, and in python form `{FieldPath("a", "b", "1.2"): value}`, and I think at some point I will need to clean up all these functions.  

But it will probably be easier to make PRs one at a time for easier reviewing and then clean up later instead of writing them all in one PR.